go.mod

```diff
@@ -155,3 +155,9 @@ require (
 	google.golang.org/protobuf v1.31.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+// replace github.com/speakeasy-api/sdk-gen-config => github.com/speakeasy-api/sdk-gen-config workflow-files
+replace github.com/speakeasy-api/sdk-gen-config => github.com/speakeasy-api/sdk-gen-config v1.5.2-0.20240109131902-459ea0b170f7
```

Comment:
Just a reminder not to forget about these

Reply:
done